application/ingestr/kafka-cmd.sh

```
@@ -0,0 +1,197 @@
+
+}
+
+function verify-data() {
+  title "Verifying data in CrateDB"
+  size_reference=5000
+  size_actual=$(
+    docker compose --file ${COMPOSE_FILE} run --rm httpie \
+      http "${CRATEDB_HTTP_URL}/_sql?pretty" stmt='SELECT COUNT(*) FROM "kafka_demo";' --ignore-stdin \
+    | jq .rows[0][0]
+  )
+  if [[ ${size_actual} = ${size_reference} ]]; then
```

Shellcheck (0.10.0)

[warning] Quote the right-hand side of = in [[ ]] to prevent glob matching.

(SC2053)